.github/workflows/release-integration.yml

```diff
@@ -0,0 +1,94 @@
+name: Release integration
+on:
+  workflow_call:
+    outputs:
+      RELEASE_TAG:
+        description: "Release Tag for Github release"
+        value: ${{ jobs.pre-release.outputs.tag-name }}
+defaults:
+  run:
+    shell: bash
+env:
+  NODE_VERSION: 14 # needed for npx
+  RELEASE_NOTES_FILE: "RELEASE-BODY.md"
+jobs:
+  pre-release:
+    runs-on: ubuntu-20.04
+    outputs:
+      tag-name: ${{ steps.create-release-package.outputs.tag-name }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Configure Git
+        run: |
+          git config user.name "GitHub Actions" # "${{ github.actor }}"
+          git config user.email "actions@github.com" # "${{ github.actor }}@users.noreply.github.com"
+
+      - name: Prepare GitHub Release Notes
+        run: |
+          # Delete pre-release tags to be able to generate a changelog from last 'real' release LOCALLY
+          # This is a workaround for a known limitation of standard-version
+          # Reference: https://github.com/conventional-changelog/standard-version/issues/203#issuecomment-872415140
+          git tag -l | grep -vE '^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$' | xargs git tag -d
+
+          # generate release notes into RELEASE_NOTES_FILE
+          npx standard-version@^9.3.1 -i "${{ env.RELEASE_NOTES_FILE }}" --skip.commit --skip.tag --header ""          
+
+      - name: Create release package
+        id: create-release-package
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "🚀 Creating release package now..."
+          # this will generate a commit and a tag
+          npx standard-version@^9.3.1 --skip.commit
+
+          TAG=$(git describe --tags --abbrev=0)
+
+          # determine tag name
+          echo "::set-output name=tag-name::$TAG"
+
+          # create a commit with the changelog
+          git add CHANGELOG.md
+          git commit -s -m "chore(release): $TAG"
```

Comment:
I always used `build(release)` for the release commit so that it doesn't show up in the changelog since the `build` annotation is ignored in the changelog.

Reply:
Fixed